bloom-housing/bloom
# fix: add the ability to disconnect community types

This PR addresses #4140 


## Description

When a reserved community type already exists on a listing and should be removed it was not properly being removed because it needs to be "disconnected" rather than just set to undefined

## How Can This Be Tested/Reviewed?

Add a community type to a listing and save the listing. Then re-edit the listing and remove the reserved community type. The community type should no longer be on the listing

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
